### ISSUE TITLE
Problems with MIMIC IV download and setup

### ISSUE THREAD

**Issue by vk002** [eth-mds/ricu#48]
Hi, 

I am trying to download MIMIC IV through ricu on a Macbook M1 2020. When I use setup_src_data() I get the following error message (below), which I understand from previous posts on the same topic that it has something to do with a new version of the dataset. Earlier replies say that the new version of ricu (0.6.0) should have fixed this, but I am still getting this error. 

```
> setup_src_data("miiv")
Error in `assert_that()`:
! hosp/caregiver.csv.gz was not found among the provided options. Did you possibly mean
icu/caregiver.csv.gz instead?
Run `rlang::last_trace()` to see where the error occurred.
```

I have also downloaded all the files and tried to import them by setting my directory to the files and using import_src(). The following code says that I seem to have imported all the tables (see below), but when I try to access the files it says "Data for miiv is missing. Setup now?". 

```
> import_src("miiv", verbose = T)
Tables `admissions`, `patients`, `transfers`, `d_hcpcs`, `diagnoses_icd`, `d_icd_diagnoses`,
`d_icd_procedures`, `d_labitems`, `drgcodes`, `emar_detail`, `emar`, `hcpcsevents`,
`labevents`, `microbiologyevents`, `pharmacy`, `poe_detail`, `poe`, `prescriptions`, …,
`provider`, and `ingredientevents` have already been imported and will be skipped
Warning message:
All required tables have already been imported 
```
```
> miiv$chartevents
Data for `miiv` is missing
Setup now (Y/n)? 
```

As you can see here, I have ricu 0.6.0, and it says that 0 of 31 tables of miiv are available, despite the above code saying that all required tables have been imported. 

```
── ricu 0.6.0 ─────────────────────────────────────────────────────────────────────────

The following data sources are configured to be attached:
(the environment variable `RICU_SRC_LOAD` controls this)

✖ miiv: 0 of 31 tables available
```

Any help would be much appreciated.

**Comment by mlondschien** [comment]
This is a bug in RICU. It's fixed in  https://github.com/eth-mds/ricu/pull/46. https://github.com/eth-mds/ricu/pull/47 is also related. You can use mimic-iv by cloning https://github.com/eth-mds/ricu/pull/46 and installing from source.

**Comment by mlondschien** [comment]
@nbenn @dplecko @prockenschaub can you merge #46 please?

**Comment by dplecko** [comment]
This issue is resolved in `ricu 0.6.1` that is now available.
